testsuite/domain/src/test/java/org/jboss/as/test/integration/domain/HostExcludesTestCase.java

```diff
@@ -130,6 +130,7 @@ private enum ExtensionConf {
                 "org.jboss.as.webservices",
                 "org.jboss.as.weld",
                 "org.jboss.as.xts",
+                "org.keycloak.keycloak-adapter-subsystem",
```

Comment:
org.keycloak.keycloak-adapter-subsystem is a new subsystem, so it should not be added to this ExtensionConf.

This ExtensionConf is the configuration for WILDFLY_10_0, and this new subsystem is added on the current WildFly release (WF27)


The approach here should be to add it to ExtensionConf.CURRENT and add the required exclusions to the domain.xml